src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -270,7 +271,7 @@ private InitCodeContext createResponseInitCodeContext(
         .initObjectType(outputType)
         .symbolTable(symbolTable)
         .suggestedName(Name.from("expected_response"))
-        .initFieldConfigStrings(context.getMethodConfig().getSampleCodeInitFields())
+        .initFieldConfigStrings(ImmutableList.<String>of())
```

Comment:
What does this change do?

Reply:
This stops using the sampleCodeInitFields when initializing the expected response object for generated tests. This is simply a bugfix - it makes no sense to use the sampleCodeInitFields, which are for the **request** object, when initializing the **response** object, which will have completely different fields.